templates/download/raspberry-pi/thank-you.html

```diff
@@ -58,6 +58,24 @@ <h2>Installation instructions</h2>
       </div>
     </div>
     <div class="row p-divider">
+      {% if version==releases.latest.core_version %}
+      <div class="col-4 blog-p-card--post">
+        <header class="blog-p-card__header--tutorial">
+          <h5 class="p-muted-heading u-no-margin--bottom">
+            core, UC20
+          </h5>
```

Comment:
These tag lists don't need to be headings, and they break the heading hierarchy - I know it's what was on the page already, but it's a quick improvement to make:
```suggestion
          <p class="p-muted-heading u-no-margin--bottom">
            core, UC20
          </p>
```